src/p11_child/p11_child_common.c

```diff
@@ -308,7 +308,7 @@ int main(int argc, const char *argv[])
 
     DEBUG_INIT(debug_level);
 
-    debug_prg_name = talloc_asprintf(NULL, "p11_child[%d]", getpid());
+    debug_prg_name = talloc_asprintf(NULL, "p11_child(%d)", getpid());
```

Comment:
Shouldn't this continue to use square brackets because it's including a process ID?

Reply:
As I see it, Journald uses a combination of `SYSLOG_IDENTIFIER` and `SYSLOG_PID` when producing the final syslog output. They get merged as string `${SYSLOG_IDENTIFIER}[${SYSLOG_PID}]: `.

So if `debug_prg_name` is set to `p11_child[1234]` then `SYSLOG_IDENTIFIER` would be set to `sssd(p11_child[1234])`, resulting in syslog output of `sssd(p11_child[1234])[1234]: `.

Potentially with a greedy regexp matching `\[(.+)\]` you get `1234])1234` as the PID.

That is the reason why I am swapping all the `debug_prg_name` square brackets to parentheses.